CI is broken for CMake MacOS

PortAudio continuous integration (GitHub Actions) appears to be [consistently failing](https://github.com/dechamps/portaudio/actions/runs/4211024589/jobs/7309086916) for "macOS Clang" on latest master (4bccdfbb68a509e2a96d916046d911959f7d935c):

```
-- Fixing pkgconfig file: /Users/runner/work/portaudio/portaudio/vcpkg/packages/jack2_x64-osx/lib/pkgconfig/jack.pc
CMake Error at scripts/cmake/vcpkg_find_acquire_program.cmake:641 (message):
  Could not find pkg-config.  Please install it via your package manager:

      brew install pkg-config
Call Stack (most recent call first):
  scripts/cmake/vcpkg_fixup_pkgconfig.cmake:198 (vcpkg_find_acquire_program)
  ports/jack2/portfile.cmake:21 (vcpkg_fixup_pkgconfig)
  scripts/ports.cmake:145 (include)


error: building jack2:x64-osx failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `git pull` and `./vcpkg update`.
Then check for known issues at:
  https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+jack2
You can submit a new issue at:
  https://github.com/microsoft/vcpkg/issues/new?template=report-package-build-failure.md&title=[jack2]+Build+error
including:
  package: jack2[core]:x64-osx -> 1.9.21
    vcpkg-tool version: 2022-03-30-692785ac944e81417840c6de244fb3e18a4b35eb
    vcpkg-scripts version: f30786c9c 2022-05-02 (10 months ago)

Additionally, attach any relevant sections from the log files above.
-- Running vcpkg install - failed
CMake Error at vcpkg/scripts/buildsystems/vcpkg.cmake:853 (message):
  vcpkg install failed.  See logs for more information:
  /Users/runner/work/portaudio/portaudio/build/vcpkg-manifest-install.log
Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.25.2/share/cmake/Modules/CMakeDetermineSystem.cmake:124 (include)
  CMakeLists.txt:2 (project)


CMake Error: CMake was unable to find a build program corresponding to "Unix Makefiles".  CMAKE_MAKE_PROGRAM is not set.  You probably need to select a different build tool.
-- Configuring incomplete, errors occurred!
CMake Error: CMAKE_C_COMPILER not set, after EnableLanguage
CMake Error: CMAKE_CXX_COMPILER not set, after EnableLanguage
Error: Process completed with exit code 1.
```

--- a/.github/workflows/cmake.yml
+++ b/.github/workflows/cmake.yml
@@ -84,6 +84,10 @@ jobs:
         sudo apt-get update
         sudo apt-get install libasound2-dev ${{ matrix.dependencies_extras }}
       if: matrix.os == 'ubuntu-latest'
+    - name: "[macOS] install dependencies"
+      # https://github.com/PortAudio/portaudio/issues/767
+      run: brew install pkg-config
+      if: matrix.os == 'macOS-latest'
     - name: "[Windows/MinGW] set up ASIO SDK cache"
       uses: actions/cache@v2
       if: matrix.asio_sdk_cache_path != null

